pkg/services/team/teamimpl/store.go

```diff
@@ -212,6 +212,13 @@ func (ss *xormStore) Search(ctx context.Context, query *team.SearchTeamsQuery) (
 			params = append(params, query.Name)
 		}
 
+		if len(query.TeamIds) > 0 {
+			sql.WriteString(` and team.id IN (?` + strings.Repeat(",?", len(query.TeamIds)-1) + ")")
```

Comment:
nice use of `strings.Repeat`

Reply:
We use this pattern a lot in SQL queries.